### ISSUE TITLE
Where is taking time from?

### ISSUE THREAD

**Issue by jackoat** [LoganMD/pwnagotchi-clock#2]
Hi,
I have installed a RTC on my pwnagotchi and it seems to be saving time and date correctly as when I connect the pwnagotchi to my computer and ssh into it the date and time are correct. When I run sudo hwclock -r the output is correct. When I took the pwnagotchi off my PC and plugged in a power source the date and time were correct. Today, after leaving the pwnagotchi plugged in overnight, the date showing is Jan 6th and the time was and hour and 20min behind.  I don't know where it is pulling this old date and time from. Is there a way to make the script pull from the hardware clock and stay up-to-date?

**Comment by LoganMD** [comment]
Oh, that's rather strange... perhaps your RTC is faulty? Mine has been working just fine since I first set it 2 years agoOn Jan 15, 2021 8:34 AM, jackoat <notifications@github.com> wrote:
Hi,
I have installed a RTC on my pwnagotchi and it seems to be saving time and date correctly as when I connect the pwnagotchi to my computer and ssh into it the date and time are correct. When I run sudo hwclock -r the output is correct. When I took the pwnagotchi off my PC and plugged in a power source the date and time were correct. Today, after leaving the pwnagotchi plugged in overnight, the date showing is Jan 6th and the time was and hour and 20min behind.  I don't know where it is pulling this old date and time from. Is there a way to make the script pull from the hardware clock and stay up-to-date?
